src/modules/billing/services/cm-refresh-service.js

```diff
@@ -96,7 +96,15 @@ const mapTransaction = (invoice, transactionMap, cmTransaction) => {
       .get(cmTransaction.id)
       .fromHash({
         isDeMinimis: cmTransaction.isDeminimis,
-        value: cmTransaction.chargeValue
+        value: cmTransaction.chargeValue,
+        calcSourceFactor: cmTransaction.calculation.WRLSChargingResponse.sourceFactor,
```

Comment:
Maybe less repetition if `WRLSChargingResponse` was destructured from `cmTransaction` first

Reply:
Totally fair. I've done this.